app/views/applications/review.html

```diff
@@ -87,7 +114,11 @@ <h1 class="govuk-heading-l">{{ title }}</h1>
         </form>
       {% else %}
 
-        {% if not (data | allSectionsCompleted) %}
+
+
+        {% if not application.personalStatement %}
+          <p class="govuk-body">You cannot submit this application until you’ve written your personal statement.</p>
+        {% elseif not (data | allSectionsCompleted) %}
 
           <p class="govuk-body">You cannot submit this application until you’ve completed your details.</p>
```

Comment:
```suggestion
          <p class="govuk-body">You cannot submit this application, it will be saved as a draft until you’ve completed your details.</p>
```